packages/cyberstorm/src/components/Dialog/Dialog.tsx

```diff
@@ -8,64 +8,32 @@ import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
 import { Tooltip } from "../..";
 import { classnames } from "../../utils/utils";
 
-type DialogProps = {
-  content?: ReactNode;
+export type DialogProps = {
```

Comment:
Why export this?

Reply:
No reason 👍

Reply:
It's still exported?

Reply:
Huh, that's weird. Should not be now. Will have to double check these in future.

Reply:
This is now needed on .stories file

Reply:
Needed in the .stories

Reply:
It's not needed, Storybook can infer the type of `args` from the information it already has? Also the import is done using the full path, which shouldn't be done, but that doesn't matter if you remove the whole import.